daffodil-io/src/main/scala/org/apache/daffodil/processors/charset/DaffodilCharsetProvider.scala

```diff
@@ -51,7 +51,8 @@ object DaffodilCharsetProvider {
     BitsCharsetUTF16LE,
     BitsCharsetUTF32BE,
     BitsCharsetUTF32LE,
-    BitsCharsetUTF8)
+    BitsCharsetUTF8,
+    BitsCharsetIBM1047 )
```

Comment:
I would prefer that this be added to this list in alphabetical order so things are a bit more organized. So move this right after BitsCharsetIBM037